include/boost/json/impl/error.ipp

```diff
@@ -49,6 +49,13 @@ case error::not_number: return "not a number";
 case error::not_exact: return "not exact";
 
 case error::test_failure: return "test failure";
+
+            case error::missing_slash: return "missing slash character";
+            case error::invalid_escape: return "invalid escape sequence";
+            case error::token_not_number: return "token is not a number";
+            case error::value_is_scalar: return "current value is scalar";
```

Comment:
Why are these indented differently from the others

Reply:
I actually thought that the old indent is a mistake. But if it's deliberate, I'll move new lines to the left.

Reply:
Not a mistake. I try to keep code narrow so it can fit in thinner places like smartphones or in an IDE window with 3 file panes. Since this code is declarative rather than imperative (i.e. it simply provides data in the form of strings and not an algorithm in the form of code), I prioritize shrinking the width measured from the first column over scope-correct indentation.